src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -62,66 +89,66 @@ export const AddressModal: React.FC<Props> = ({
               type="text"
               title="Full Name"
               onChange={formik.handleChange}
-              error={formik.errors.name}
-              value={formik.values.name}
+              error={formik.errors?.name}
```

Comment:
changing these error props to:
```tsx
onBlur={formik.handleBlur}
error={formik.touched.name && formik.errors.name}
```
The field wouldn't show a red error message before the user editing the field

Reply:
You mean I should change these? I'm seeing red error msgs when  required fields are not populated, which is what we want I believe.

Reply:
In the screenshot above I can see the address line 1, city etc. get red as soon as you enter the first name. The error should show up only after the user edits that. 👆🏽 will fix that.